CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 5.8.1 (unreleased)
 ------------------
 
-- Use custom creator criteria.
+- Use custom creator criteria to list only creators present in the catalog.
```

Comment:
Why do you need to list only the creators in the catalog instead of the original one?